src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEntryPointSymbol.cs

```diff
@@ -361,43 +360,37 @@ private enum ParamKind
             // Task<_> -> Int32
             private static TypeSymbol TranslateReturnType(CSharpCompilation compilation, TypeSymbol returnType)
             {
-                if (returnType.IsGenericTaskType(compilation)) {
-
-                    return compilation.GetSpecialType(SpecialType.System_Int32);
-                }
-                else
-                {
-                    return compilation.GetSpecialType(SpecialType.System_Void);
-                }
+                return returnType.IsGenericTaskType(compilation)
+                    ? compilation.GetSpecialType(SpecialType.System_Int32)
+                    : compilation.GetSpecialType(SpecialType.System_Void);
```

Comment:
Is it possible for `GetSpecialType` to return `null` here or are we sufficiently protected by this point?  If it can return null then we need to account for that as the return here is used in places where `null` isn't acceptable.